src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraDto.kt

```diff
@@ -33,19 +35,18 @@ data class Entry(
 )
 
 @Serializable
-data class ImageData(
+class ImageData(
     val id: Int,
     val key: String,
     val hash: String,
 )
 
 @Serializable
-data class EntryKey(
+class EntryKey(
     val id: Int,
     val key: String? = null,
     val hash: String? = null,
     val url: String? = null,
-    val names: List<String> = emptyList(),
```

Comment:
Isn't this needed for the gallery-data list?

Reply:
It isn't needed anymore since the image names are now included in the gallery response.